stockflux-securities-master/src/components/InputForm.css

```diff
@@ -9,6 +9,7 @@
 .input-form-body {
   display: flex;
   flex-direction: column;
+  height: 300px;
```

Comment:
Is there a reason for hard-coding the height of the form to 300px? Is there not a variable somewhere that defines this height?

Reply:
There is not a variable that defines the height although I could add one

Reply:
I've added a css variable for this height now